custom_components/lennoxs30/sensor.py

```diff
@@ -47,6 +49,10 @@ async def async_setup_platform(
         _LOGGER.info(f"Create S30OutdoorTempSensor sensor system [{system.sysId}]")
         sensor = S30OutdoorTempSensor(hass, manager, system)
         sensor_list.append(sensor)
+        if system.diagInverterInputVoltage != None:
```

Comment:
Let’s add a config option to enable.    Not all HVACs have inverters just the variable speed ones, so this sensor will only be relevant for those users.  Also, as a preference, I like an opt-in model prior to creating new sensors.

Reply:
Can do!